app/models/account_request.rb

```diff
@@ -21,6 +21,7 @@ class AccountRequest < ApplicationRecord
   validates :email, presence: true, uniqueness: true
   validates :request_details, presence: true, length: { minimum: 50 }
   validates :email, format: { with: URI::MailTo::EMAIL_REGEXP }
+  validates :organization_website, format: { with: URI::DEFAULT_PARSER.make_regexp, message: "it should look like 'http://www.example.com'" }, allow_blank: true
```

Comment:
Please remove the "it" at the beginning of the message.   Thanks.

Reply:
Removed the "it" from the message